src/Microsoft.Identity.Web.TokenCache/Distributed/MsalDistributedTokenCacheAdapter.cs

```diff
@@ -233,22 +233,14 @@ protected override async Task WriteCacheBytesAsync(string cacheKey, byte[] bytes
         {
             string write = "Write";
```

Comment:
not related: since this is in the critical path for perf / mem usage, avoid creating new objects. In this case, just use a constant.